Makefile

```diff
@@ -30,14 +79,17 @@ version:
 	@echo $(VERSION)
 
 ./bin/node-problem-detector: $(PKG_SOURCES)
-	GOOS=linux go build -o bin/node-problem-detector \
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux go build -o bin/node-problem-detector \
 	     -ldflags '-w -extldflags "-static" -X $(PKG)/pkg/version.version=$(VERSION)' \
```

Comment:
As per the comment above the "-static" here looks fishy, and probably should only be here when not building with journald support?